docs/guides/deploy.md

```diff
@@ -131,3 +131,64 @@ Then, decrypt the result:
 decrypted_result = client.decrypt(deserialized_result)
 assert decrypted_result == 49
 ```
+
+# Deployment of modules
+
+Deploying a [module](../compilation/modules.md#modules) follows the same logic presented above. Assuming a module compiled in the following way:
```

Comment:
First sentence: Deploying a [module](../compilation/modules.md#modules) follows the same logic.